### ISSUE TITLE
Good Page Titles

### ISSUE THREAD

**Issue by cawlin** [evangillespie/monsterCards#11]
We should have good page titles that work well for screen readers. 

`<level 2> - <level 1>`

* Monsters - D&D 5E SRD
* Aboleth - D&D 5E SRD
